dashboard/app.py

```diff
@@ -329,9 +333,13 @@ def filter_by_vendor_location(df, selected_country, selected_state, selected_cou
 
 # st.altair_chart(chart, use_container_width=True)
 
+# df = df.style.format({
+#     "amount_cancelled": "${:,.2f}"
+# })
 
-st.dataframe(df)
-
+st.dataframe(df, column_config={
```

Comment:
Confirmed I'm seeing the currency formatting in this branch, then when I switch to main, the formatting is not present.